app/forms/idv/phone_form.rb

```diff
@@ -4,21 +4,21 @@ class PhoneForm
     include FormPhoneValidator
 
     attr_reader :idv_params, :user, :phone
+    attr_accessor :international_code
 
     def initialize(idv_params, user)
       @idv_params = idv_params
       @user = user
       self.phone = (idv_params[:phone] || user.phone).phony_formatted(
         format: :international, normalize: :US, spaces: ' '
       )
+      self.international_code = 'US'
```

Comment:
should we pull out a constant like `PhoneFormatter::DEFAULT_COUNTRY` or something here? We have a few random hardcoded magic constants of 'US' for things like this